docs/reference/integrations.md

```diff
@@ -0,0 +1,66 @@
+# Integrations
+
+### db
+
+_Interface_: mysql  
+_Supported charms_: [charmed-osm-mariadb-k8s](https://charmhub.io/mysql),
+[mysql-k8s](https://charmhub.io/mysql-k8s)
+
+Database integration is a required relation for the wordpress-k8s charm to supply structured data
+storage for WordPress. It is recommended to use a juju native integration that provides a mysql
+interface by providing necessary configuration details to wordpress-k8s charm. See
```

Comment:
Would it be worth including more information about which are the necessary configuration options in the case of using an integration?

Reply:
What about an example for how to configure the integration?

Reply:
Added `mysql-interface-user` and `mysql-interface-database` parameter configuration for mysql interface.

On the second note about example on how to configure the integration, I think examples are better suited in how-to guide or in the tutorial, which should be ready for review soon :)